### PR TITLE
Display yuki correctly in /ssb

### PR DESCRIPTION
**Pull request by AnnikaCodes**

### CODE REVIEW

File: data/mods/ssb/random-teams.ts [hunk 1]
```diff
@@ -427,6 +427,41 @@ export const ssbSets: SSBSets = {
 		signatureMove: 'Class Change',
 		evs: {hp: 4, spa: 252, spe: 252}, ivs: {atk: 0}, nature: ['Modest', 'Timid'],
 	},
+	'yuki-Cleric': {
+		species: 'Pikachu-PhD', ability: 'Triage', item: 'Light Ball', gender: 'F',
+		moves: ['Parabolic Charge', 'Wish', 'Baton Pass'],
+		signatureMove: 'Class Change',
+		evs: {hp: 4, spa: 252, spe: 252}, ivs: {atk: 0}, nature: ['Modest', 'Timid'],
```
Comment (ghost) [inline data/mods/ssb/random-teams.ts]: Honestly I think in this scenario the nature  should remain constant

Comment (AnnikaCodes) [reply data/mods/ssb/random-teams.ts]: Well, the `nature` attribute is a mandatory part of an `SSBSet`, so something has to be specified. However, I can make `changeSet` not modify the nature when the set is yuki.

Comment (ghost) [reply data/mods/ssb/random-teams.ts]: i can figure something out

File: data/mods/ssb/abilities.ts [hunk 2]
```diff
@@ -1,5 +1,71 @@
...
+		hp: newSet.ivs?.hp || 31,
+		atk: newSet.ivs?.atk || 31,
+		def: newSet.ivs?.def || 31,
+		spa: newSet.ivs?.spa || 31,
+		spd: newSet.ivs?.spd || 31,
+		spe: newSet.ivs?.spe || 31,
+	};
+	pokemon.set.evs = evs;
+	pokemon.set.ivs = ivs;
+	pokemon.set.nature = Array.isArray(newSet.nature) ? context.sample(newSet.nature) : newSet.nature;
```
Comment (ghost) [inline data/mods/ssb/abilities.ts]:
```suggestion
	if (newSet.nature) pokemon.set.nature = Array.isArray(newSet.nature) ? context.sample(newSet.nature) : newSet.nature;
```

File: server/chat-plugins/random-battles.ts [hunk 3]
```diff
@@ -325,7 +325,7 @@ function SSBSets(target: string) {
 			}
 			buf += `<li>EVs: ${evs.join(" / ")}</li>`;
 		}
-		buf += `<li>${Array.isArray(set.nature) ? set.nature.join(" / ") : formatNature(set.nature)} Nature</li>`;
+		if (set.nature) buf += `<li>${Array.isArray(set.nature) ? set.nature.join(" / ") : formatNature(set.nature)} Nature</li>`;
```
Comment (ghost) [inline server/chat-plugins/random-battles.ts]:
```suggestion
		if (set.nature) {
			buf += `<li>${Array.isArray(set.nature) ? set.nature.join(" / ") : formatNature(set.nature)} Nature</li>`;
		}
```

**Pull request merged**